indico/modules/events/registration/util.py

```diff
@@ -377,3 +377,41 @@ def get_events_registered(user, from_dt=None, to_dt=None):
              .filter(Registration.is_active, ~RegistrationForm.is_deleted, ~Event.is_deleted)
              .filter(event_date_filter))
     return {registration.event_id for registration in query}
+
+
+def build_registrations_api_data(event):
+    if not event.registration_forms:
```

Comment:
This never succeeds since `event.registration_forms` is a dynamic relationship and the query object don't implement a `__nonzero__` or `__bool__` magic method to check if they'd return any rows. So this check can simply go away